ApollosProject/apollos-prototype
# Updates typographic styles

## DESCRIPTION
This PR changes the apps vertical rhythm.

### What does this PR do, or why is it needed?
Previously we used a `baseLineHeight` of `23.04` which was derived by a `baseFontSize` of `16` with a vertical Rhythm (read: ratio) of `1.44`. This PR changes the `baseLineHeight` to `24` creating a perfect fifth ratio of `1.5` or `2:3`. Aside from being what @kevin-birdy wanted it also is more likely to produce whole numbers and that makes him feel warm and fuzzy inside.

**Note:** the `lineHeight` on `H1`, `H2`, and `H3` are unchanged and continue to use a ratio of `1.15` or `20:23`. The `margin/padding` on these components **has changed** to reflect the new `1.5` everything else uses.

### What design trade-offs/decisions were made?
No trade offs but this does "loosen" the app a bit. All typographic elements will have a touch more `lineHeight` and vertical `padding/margins`. This also increases the paragraph spacing.

### How do I test this PR?
There are a TON of snapshot changes here. My request is to just have @kevin-birdy give this branch a look over and make sure he is happy with the changes. Then an engineer can just review that everything works as expected. Given that this is entirely a frontend UI change I'm not even sure it's necessary for an engineer to review this.

### What automated tests did you write?
Existing snapshot coverage.

## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
